contrib/tracing/README.md

```diff
@@ -43,3 +43,88 @@ example, to point to release builds if needed. See the
 kernel VM. This means the bpftrace and BCC examples must be executed with root
 privileges. Make sure to carefully review any scripts that you run with root
 privileges first!**
+
+### log_p2p_traffic.bt
+
+A bpftrace script logging information about inbound and outbound P2P network
+messages. Based on the `net:inbound_message` and `net:outbound_message`
```

Comment:
Wishlist: this for disconnection and eviction events, with the message, connection direction and type, peer id and address.

Reply:
Agree! We discussed something similar in https://github.com/bitcoin/bitcoin/issues/20981#issuecomment-766096054 too. Thanks for the input on the arguments! I'll resolve this in favor of https://github.com/bitcoin/bitcoin/issues/20981#issuecomment-845833323 :)

Reply:
(could add the network the peer connected through to the args list)

Reply:
@jonatack  there's https://github.com/bitcoin/bitcoin/pull/25832 now!